overlay-memfault.conf

```diff
@@ -6,6 +6,7 @@
 
 CONFIG_MEMFAULT=y
 CONFIG_MEMFAULT_NCS_PROJECT_KEY=""
+CONFIG_MEMFAULT_NCS_DEVICE_ID_IMEI=y
```

Comment:
Note this is already the default, nordic just added to make sure their CI built: https://github.com/memfault/sdk-nrf/commit/489a528d7fcc46982377822448c37ce4ad55309f